content/en/docs/tasks/security/automtls/index.md

```diff
@@ -0,0 +1,289 @@
+
+{{< text bash >}}
+$ istioctl manifest apply --set profile=demo \
+  --set values.global.mtls.auto=true \
+  --set values.global.mtls.enabled=false
+{{< /text >}}
+
+## Instructions
+
+### Setup
+
+Our examples deploy `httpbin` service into three namespaces, `full`, `partial`, and `legacy`.
+Each represents different phase of Istio migration.
+
+`full` namespace contains all server workloads finishing the Istio migration. All deployments have
+sidecar injected.
+
+{{< text bash >}}
+$ kubectl create ns full
+$ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n full
+$ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n full
+{{< /text >}}
+
+`partial` namespace contains server workloads partially migrated to Istio. Only migrated one has
+sidecar injected, able to serve mutual TLS traffic.
+
+{{< text bash >}}
+$ kubectl create ns partial
+$ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n partial
+$ cat <<EOF | kubectl apply -n partial -f -
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: httpbin-nosidecar
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: httpbin
+        version: nosidecar
+    spec:
+      containers:
+      - image: docker.io/kennethreitz/httpbin
+        imagePullPolicy: IfNotPresent
+        name: httpbin
+        ports:
+        - containerPort: 80
+EOF
+{{< /text >}}
+
+`legacy` namespace contains the workloads and none of them have Envoy sidecar.
+
+{{< text bash >}}
+$ kubectl create ns legacy
+$ kubectl apply -f @samples/httpbin/httpbin.yaml@ -n legacy
+$ kubectl apply -f @samples/sleep/sleep.yaml@ -n legacy
+{{< /text >}}
+
+Last we deploy two `sleep` workloads, one has sidecar and one does not.
+
+{{< text bash >}}
+$ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n full
+$ kubectl apply -f @samples/sleep/sleep.yaml@ -n legacy
+{{< /text >}}
+
+You can confirm the deployments in all namespaces.
+
+{{< text bash >}}
+$ kubectl get pods -n full
+$ kubectl get pods -n partial
+$ kubectl get pods -n legacy
+NAME                      READY   STATUS    RESTARTS   AGE
+httpbin-dcd949489-5cndk   2/2     Running   0          39s
+sleep-58d6644d44-gb55j    2/2     Running   0          38s
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-6f6fc94fb6-8d62h   1/1     Running   0          10s
+httpbin-dcd949489-5fsbs    2/2     Running   0          12s
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-54f5bb4957-lzxlg   1/1     Running   0          6s
+sleep-74564b477b-vb6h4     1/1     Running   0          4s
+{{< /text >}}
+
+You should also verify that there is a default mesh authentication policy in the system, which you can do as follows:
+
+{{< text bash >}}
+$ kubectl get policies.authentication.istio.io --all-namespaces
```

Comment:
Not sure if we need to mention this. It is internal policy for grafana and may confuse users.

Reply:
This is copied from old doc. I think it ensure that customer dont have some leftover of their own policy, right?